scripts/CIP-TOWER.ts

```diff
@@ -0,0 +1,87 @@
+// run with
+// npx hardhat run ./scripts/CIP-TOWER.ts --network hardhat
+import hre, {deployments, network, hardhatArguments} from "hardhat";
+import {castVote, createProposal, executeProposal, fundMultisign, queueProposal} from "./utils";
+import {BigNumber} from "ethers";
+
+async function main() {
+	const ethers = hre.ethers;
+
+	let orchestratorAddress = "0x373C74BcE7893097ab26d22f05691907D4f2c18e";
+	let tcap = await deployments.get("TCAP");
+	//TODO: waiting for real deployment
+	let tcapContract = await ethers.getContractAt("TCAP", tcap.address);
+	let usdcHardVault = await deployments.get("AaveVaultHandler");
```

Comment:
If we are going with hardcoded values, can we use hardcoded values here?

Reply:
Yes! left it like that for now as contracts are not deployed